testing/rundeck/APKBUILD

```diff
@@ -0,0 +1,86 @@
+# Contributor: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+# Maintainer: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+pkgname=rundeck
+pkgver=2.6.9
+pkgrel=0
+pkgdesc="Job scheduler and runbook automation. Enable self-service access to existing scripts and tools."
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+url="http://rundeck.org"
+arch="noarch"
+license="ASL 2.0"
+depends="openjdk8 bash ncurses5-libs"
+depends_dev=""
+makedepends="$depends_dev"
+install="$pkgname.pre-install"
+subpackages=""
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/rundeck/${pkgname}/archive/v${pkgver}.tar.gz
+	rundeck.initd
+	rundeck.confd
+	"
+builddir="$srcdir"/$pkgname-$pkgver
+JVERS=$pkgver-SNAPSHOT
+
+build() {
+	cd "$builddir"
+	export JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk
+	export TERM=dumb
+	bash -c "$builddir/gradlew build"
+}
+
+package() {
+	cd "$builddir"
+	install -m755 -D "$srcdir"/$pkgname.initd \
+		"$pkgdir"/etc/init.d/$pkgname || return 1
+	install -m644 -D "$srcdir"/$pkgname.confd \
+		"$pkgdir"/etc/conf.d/$pkgname || return 1
+	install -d -o $pkgusers -g $pkggroups \
+		"$pkgdir"/usr/bin \
+		"$pkgdir"/var/lib/rundeck/.ssh \
+		"$pkgdir"/var/lib/rundeck/exp/webapp \
+		"$pkgdir"/var/lib/rundeck/data \
+		"$pkgdir"/var/lib/rundeck/work \
+		"$pkgdir"/var/lib/rundeck/logs \
+		"$pkgdir"/var/lib/rundeck/cli \
+		"$pkgdir"/var/lib/rundeck/libext \
+		"$pkgdir"/var/lib/rundeck/bootstrap \
+		"$pkgdir"/var/lib/rundeck/libext/cache \
+		"$pkgdir"/var/lib/rundeck/var \
+		"$pkgdir"/var/lib/rundeck/var/tmp/pluginJars \
+		"$pkgdir"/var/log/rundeck \
+		"$pkgdir"/tmp/rundeck \
+		"$pkgdir"/var/rundeck/projects \
+		"$pkgdir"/etc/rundeck || return 1
+	cp -R core/build/tools/lib/*.jar \
+	   "$pkgdir"/var/lib/rundeck/cli || return 1
+	cp core/build/libs/rundeck-core-"$JVERS".jar \
+	   "$pkgdir"/var/lib/rundeck/cli || return 1
+	cp rundeck-launcher/rundeck-jetty-server/build/libs/rundeck-jetty-server-"$JVERS".jar \
+	   "$pkgdir"/var/lib/rundeck/bootstrap || return 1
+	cp rundeck-launcher/launcher/build/launcher-contents/lib/*.jar \
+	   "$pkgdir"/var/lib/rundeck/bootstrap/ || return 1
```

Comment:
This directory contains `jna-3.2.2.jar` that bundles a native library (`libjnidispatch.so`), compiled against glibc, so it _may_ not work on Alpine correctly. Moreover, this makes the package arch-dependent and limits it only to x86 and x86_64.

There’s a package [java-jna](https://pkgs.alpinelinux.org/package/edge/community/x86_64/java-jna), compiled for all archs, but Rundeck still uses old 3.x version. However, I hope that it may not be a problem to create a package for this old version.

This JAR is also in `/var/lib/rundeck/cli` and `/var/lib/rundeck/exp/webapp/WEB-INF/lib`, but in newer version 4.1.0.

You can read [this article](https://github.com/jruby/jruby/wiki/JRuby-on-Alpine-Linux) for more information about this topic.